spec/spec_helper.rb

```diff
@@ -3,7 +3,7 @@
 require 'openstates'
 require 'pry'
 require 'pry-remote'
-require 'pry-debugger'
+require 'pry-byebug'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.